src/firmware_new/boards/frankie_v1/io/drivetrain.c

```diff
@@ -0,0 +1,48 @@
+#include "firmware_new/boards/frankie_v1/io/drivetrain.h"
+
+#include <assert.h>
+#include <stdbool.h>
+
+static DriveTrainUnit_t *_front_left_drive_unit;
+static DriveTrainUnit_t *_front_right_drive_unit;
+static DriveTrainUnit_t *_back_left_drive_unit;
+static DriveTrainUnit_t *_back_right_drive_unit;
```

Comment:
Nit:

Don't we usually keep the '*' on the type?

Reply:
Hmm, I definitely prefer that, but this is also definitely an automated formatting thing, so I don't care to much since we're being consistent.